### PR TITLE
[WNMGDS-2042] Refactor token exporter

### PR DESCRIPTION
**Pull request by zarahzachz**
## Summary

WNMGDS-2042

- Refactor `css-vars` command to export all tokens
- `css-vars` generates Sass map for critical Sass vars
- Remove `sass` from export option, delete Sass exporter file

### CODE REVIEW

File: packages/design-system/src/styles/settings/variables/_core-scss-to-css.map.scss [hunk 1]
```diff
@@ -0,0 +1,15 @@
+$grid-columns: 12 !default;
```
Comment (forrestbaer) [inline packages/design-system/src/styles/settings/variables/_core-scss-to-css.map.scss:1]: maybe these filenames could be something more like `layout-tokens.scss` ? The map isn't really relevant anymore with these.

File: packages/design-system-tokens/src/lib/file.ts [hunk 2]
```diff
@@ -27,7 +27,7 @@ export const getAllFiles = (rootPath: string, arrayOfFiles: string[] = []): stri
 };
 
 /**
- * Search out all availible modules under a path and return an
+ * Search out all available modules under a path and return an
```
Comment (forrestbaer) [inline packages/design-system-tokens/src/lib/file.ts:30]: 🙌🏻

**Review by forrestbaer — COMMENTED** [review]
Looks pretty good outside of a few small comments!

**Review by pwolfert — APPROVED** [review]
I just noticed that our `yarn build:tokens` command at the root needs to be updated, but other than that LGTM 🎉

**Pull request merged**